### PR TITLE
Fried Eggs Now Grillable

### PR DESCRIPTION
**Pull request by ThisFlux**
## About The Pull Request

Made the egg grillable and the fried egg burn slowly instead of instantly. Removed the old crafting recipe for fried eggs.

## Why It's Good For The Game

One time I was cooking in front of a Plasma-man. He wanted to learn more about food and I wanted to make a fried egg. Naturally, I threw an egg onto the grill. It caught fire and the Plasma-man was like "is that supposed to happen?". I had to claim I was being stylish, but I don't think he was convinced. This fix will prevent this embarrassment from happening to anyone else and also makes more sense than magically cooking eggs with salt and pepper.

## Changelog
:cl: Flux
add: Fry eggs on the grill
del: Removed old fried egg crafting recipe
/:cl:

### CODE REVIEW

**Comment by maxymax13** [comment]
nooooo how will i stop walking at 1cm/s when the chef is busy doing god knows what if i can't alchemize eggs into being fried with pepper and salt??

**Comment by jughu** [comment]
from microwaving eggs to turn em into boiled eggs, then into fried eggs.
to alchemizing egg with pepper and salt into a fried egg.
we have finally come to the true result of the egg

File: code/game/objects/items/food/egg.dm [hunk 1]
```diff
@@ -85,13 +85,17 @@
 /obj/item/food/egg/yellow
 	icon_state = "egg-yellow"
 
+/obj/item/food/egg/MakeGrillable()
```
Comment (AMonkeyThatCodes) [inline code/game/objects/items/food/egg.dm:88]: this should work like how pancakes work. Putting an egg ontop of a grill is not how you make a fried egg.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
@AMonkeyThatCodes is right, see how `/datum/chemical_reaction/food/pancakebatter` is setup and used.

This should be made with eggyolk and eggwhites.

**Comment by Cyberboss** [comment]
Superseded by #61259

**Pull request closed without merging**